core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -455,7 +455,13 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
       .defaultsTo(1)
 
     val whitelistOpt = parser.accepts("whitelist",
-      "Whitelist of topics to mirror.")
+      "DEPRECATED, use --include instead; ignored if --include specified. List of included topics to mirror.")
```

Comment:
The term "included" seems redundant ```suggestion
      "DEPRECATED, use --include instead; ignored if --include specified. List of topics to mirror.")
```